components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIProviderHostObject.java

```diff
@@ -579,6 +579,7 @@ public static boolean jsFunction_manageAPI(Context cx, Scriptable thisObj, Objec
 
             //Save swagger in the registry
             apiProvider.saveSwagger20Definition(api.getId(), (String) apiData.get("swagger", apiData));
+            apiProvider.addSwaggerToLocalEntry(api, (String) apiData.get("swagger", apiData));
```

Comment:
Shall we move "swagger" to a string constant?

Reply:
Updated